tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -3,6 +3,7 @@
 # generated. These should be annotated with a comment like
 # # stubgen override
 # to help the sanity of maintainers.
+import typing
```

Comment:
What is the reason we import typing again when we already imported Optional and others separately?

Reply:
This is how stubgen generated it, I'm trying to avoid needlessly tweaking the output